folium/plugins/timestamped_wmstilelayer.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+TimestampedGeoJson plugin
+--------------
+
+Add a timestamped geojson feature collection on a folium map.
+This is based on Leaflet.TimeDimension.
+
+https://github.com/socib/Leaflet.TimeDimension
+
+A geo-json is timestamped if:
+    * it contains only features of types LineString, MultiPoint,
+      MultiLineString and MultiPolygon.
+    * each feature has a "times" property with the same length as the
+      coordinates array.
+    * each element of each "times" property is a timestamp in ms since epoch,
+     or in ISO string.  Eventually, you may have Point features with a
+     "times" property being an array of length 1.
+
+"""
+
+from jinja2 import Template
+
+from branca.element import Figure, JavascriptLink, CssLink
+from ..map import Layer
+from ..features import WmsTileLayer
+
+
+class TimestampedWmsTileLayers(Layer):
+    def __init__(self, data, transition_time=200, loop=False, auto_play=False,
+                 period="P1D", time_interval=False):
```

Comment:
In folium we prefer single quotes `'` instead of double `"` to make it clear to ready JSON and Python. (JSON are only double.)